crates/astria-composer/src/executor/tests.rs

```diff
@@ -183,6 +186,53 @@ async fn mount_broadcast_tx_sync_seq_actions_mock(server: &MockServer) -> MockGu
         .await
 }
 
+/// Mounts a `CometBFT` status response with a specified mock sequencer chain id
+async fn mount_genesis(server: &MockServer, mock_sequencer_chain_id: &str) {
```

Comment:
For a followup: Sucks that we need this here. We should just rip out all of these wrongly placed pseudo-blackbox tests and just rely on `tests/blackbox`.

Reply:
Would this be an issue for later or would it be better to do that in this PR?

Reply:
No, that's why I prefixed it with "for a followup" :-)

Reply:
Ah, I see. I was confused as to whether it meant followup in this PR or not. Thanks for the clarification! I also changed the comment in 8523e04 to reflect now mounting the genesis as opposed to a status response.